homeassistant/components/remote_calendar/strings.json

```diff
@@ -18,14 +21,14 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
     },
     "error": {
-      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "cannot_connect": "[%key:common::config_flow::error::timeout_connect%]",
```

Comment:
To clarify, what I meant was to the existing case alone., then add a separate except block for `TimeoutException` and another `timeout_connect` entry here.

Reply:
okay, this was done here: https://github.com/home-assistant/core/pull/147024/commits/21a6d54bb48783883d8bbd13df8a0e5b4c83f45d.
I don't understand your philosophy. Should this also be done like that in the coordinator?

Reply:
Done in https://github.com/home-assistant/core/pull/147024/commits/c0c2e51a6cd0d8e3b52be8c788b6d609402f05ea